### ISSUE TITLE
Personaje atascado en isla bruma azur

### ISSUE THREAD

**Issue by sasorielmago** [RoDNet/Bugtracker-Infinium-4.0.6a#1042]
Buenas a todos bueno la casa trata de que estoy haciendo los logros del wow y al descubrir la zona de la isla bruma azur el personaje se queda atascado y me sale el wow error y no puedo hacer nada ya me paso 1 vez y el personaje se murió pero ahora esta sobre montura y no puedo hacer nada. alguna solucion?

**Comment by Alfonn** [comment]
Sí, a mi me pasó igual, en la zona oeste hay 1 sitio dónde no deja pasar, cada vez que lo intentas o logueas te sale el wowerror, a mí se me arregló solo, entre un par de veces y alfinal acabó dejandome.-

Añado info.
http://es.wowhead.com/zone=3524/isla-bruma-azur

Encima de dónde están la cueva con las 3 casas, por ahí es, Un saludo. Atte:froosty
